### PR TITLE
Get json instead of text from htmlhint

### PR DESCRIPTION
**Pull request by prosf**
linter-htmlhint doesn't work anymore as yaniswang/HTMLHint has changed its default text output

For example 
  output before:
    line 2, col 1: Special characters must be escaped : [ < ].
  output now:
          L2 |<lang="en">
                ^ Special characters must be escaped : [ < ]. (spec-char-escape)

This branch uses the Json output that yaniswang/HTMLHint offers and so avoids the more costly regex parsing.

### CODE REVIEW

File: lib/init.coffee [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports =
       @subscriptions.add atom.config.observe 'linter-htmlhint.executablePath',
         (executablePath) =>
           @executablePath = executablePath
-      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.handlebars', 'text.html.php', 'text.html.ruby']
+      @scopes =  ['text.html.angular', 'text.html.basic', 'text.html.erb', 'text.html.gohtml', 'text.html.jsp', 'text.html.mustache', 'text.html.php', 'text.html.ruby']
```
Comment (Arcanemagus) [inline lib/init.coffee]: Did this not support `text.html.handlebars` properly in the first place, or why did you remove that?

Comment (prosf) [inline lib/init.coffee]: Sorry I missed that as I did copy paste from the corresponding init.coffee
that is installed on my Atom Editor.
It's strange ,though , as I did a clean install yesterday of the Atom and
its .atom folder.

So please add 'text.html.handlebars' back. :)
Thanks

Comment (Arcanemagus) [inline lib/init.coffee]: This is your PR, nobody can touch it unless this gets merged in by one of the package maintainers :wink:.

**Comment by thearabbit** [comment]
I have the same problem.
I use `Blaze in Meteor JS`.

**Comment by Arcanemagus** [comment]
@thearabbit: @prosf closed this without addressing the comments, this would need to be re-implemented, PR's are always welcome!
